### DISCUSSION TITLE
Espresenses rebooting, serious update issues.

### DISCUSSION THREAD

**Discussion by Wheemer** [ESPresense/ESPresense#825] [Support]
Hello;

I had 4 of these setup around my home. Recently they have been constantly rebooting. They were all on v3.2.0b8 with Automatic updates on and prerelease checked. However they are not updating, or perhaps that is the reason they are rebooting constantly?

I noticed a new version v3.2.0b10 so I placed the url https://github.com/ESPresense/ESPresense/releases/download/v3.2.0b10/esp32.bin in the update url and rebooted.

The device never came back and is now not recognized by the pc.

Any help as to how to actually update without bricking would be great.

Advice on if this esp32 is recoverable would really help also.

Thanks
Steve Reid

**Comment by colinhatfield** [comment]
I was just about to post, exact same situation here.

Thanks
Colin

**Comment by Wheemer** [comment]
I have disabled updates and the remaining 3 have stopped constantly rebooting.

However it seems as if the one I manually updated is completely dead. When I plug it into the pc a tiny chip behind the usb gets super hot. I also tried to connect to it via a usb to ttl and I get nothing at all. Just a red light without serial communications.

**Comment by Wheemer** [comment]
I have updated my remaining 4 esp32s to 3.2.1 manually without an issue.

However updating the one sensor that was on b8 via the url for b10 has most definitely bricked it. The serial IC is burning hot now.

**Comment by DTTerastar** [comment]
v3.2.0b8 had a bad issue.  You can work around it by using esptool and turning on Arduino OTA

**Reply by Wheemer** [reply]
The device no longer appears when connecting to the pc, and doesn't come online at all. I have already ordered a replacement.

**Reply by DTTerastar** [reply]
I doubt the bad firmware could do that, but ESPresense in general runs the CPU pretty hard and does seem to wear out weak ESP32's.

**Comment by Wheemer** [comment]
I would consider this a weak one, it looks lower build quality than the others I have. However I did not risk the forced web update on my remaining esp32s. I plugged them in and flashed as new again to be safe.

The manual update via pasted url to the b10 beta did indeed fry it though. I waited 30 mins for it to come back before unplugging and connecting to the pc only to not be able to get a com port or connection sound ever again. I even hooked it up to a usb to ttl to see if I could communicate with it, but nothing worked again.

**Comment by DanielTerletzkiy** [comment]
same issue, with fresh install of the newest version. very disappointed to have my esp32c3 bricked, by a faulty version that still exists to this date
